docs/tutorials/chisel.rst

```diff
@@ -96,49 +93,12 @@ Now you can run a container from the rock:
     :end-before: [docs:docker-run-end]
     :dedent: 2
 
-The output will be OpenSSL's default help message, which starts like this:
+Which should print:
 
 ..  code-block:: text
-    :class: log-snippets
-
-    help:
-
-    Standard commands
-    asn1parse         ca                ciphers           cmp
-    cms               crl               crl2pkcs7         dgst
-    dhparam           dsa               dsaparam          ec
-    ecparam           enc               engine            errstr
-    fipsinstall       gendsa            genpkey           genrsa
-    help              info              kdf               list
-    mac               nseq              ocsp              passwd
-    pkcs12            pkcs7             pkcs8             pkey
-    pkeyparam         pkeyutl           prime             rand
-    rehash            req               rsa               rsautl
-    s_client          s_server          s_time            sess_id
-    <... many more lines of output>
-
-As you can see, OpenSSL has many features. Use one of them to check that
-Ubuntu's website has valid SSL certificates:
+    :class: foo
```

Comment:
why the class change?

Reply:
It's the same one used in hello-world rock tutorial. The whole output of openssl example is removed and I put the hello-world output in its place.

Reply:
but is class `foo` defined somewhere? or is it just a leftover from testing?

Reply:
It is used here: https://github.com/canonical/rockcraft/blob/e42d8df9963b6e660ae0df9ca6276901541e639a/docs/tutorials/hello-world.rst?plain=1#L100
It could be a leftover.

Reply:
used, yes, but is it defined? cause if not, then it would have not impact to remove it

Reply:
changed it as it's not defined anywhere.